### PR TITLE
merge changes from main to develop branch

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: extras/openvino-operator-openshift/openshift-bundle/0.2.0/Dockerfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
 LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
-LABEL com.redhat.openshift.versions="v4.6,v4.7"
```
Comment (atobiszei) [inline extras/openvino-operator-openshift/openshift-bundle/0.2.0/Dockerfile:34]: Should we narrow ourselves to 4.6 only on develop or is it just temporary for release?

**Review by atobiszei — APPROVED** [review]

File: extras/openvino-operator-openshift/openshift-bundle/0.2.0/manifests/openvino-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -226,7 +227,7 @@ spec:
                   - --metrics-bind-address=127.0.0.1:8080
                   - --leader-elect
                   - --leader-election-id=ovms-operator
-                image: registry.connect.redhat.com/intel/openvino-operator:0.2.0
+                image: registry.connect.redhat.com/intel/ovms-operator:0.2.0
```
Comment (mzegla) [inline extras/openvino-operator-openshift/openshift-bundle/0.2.0/manifests/openvino-operator.clusterserviceversion.yaml:230]: Is this change intentional? Shouldn't it be `openvino-operator`?

File: extras/openvino-operator-openshift/openshift-bundle/0.2.0/metadata/annotations.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: openvino-operator
+  operators.operatorframework.io.bundle.package.v1: ovms-operator
```
Comment (mzegla) [inline extras/openvino-operator-openshift/openshift-bundle/0.2.0/metadata/annotations.yaml:6]: Is this change intentional? Shouldn't it be `openvino-operator`?

File: extras/openvino-operator-openshift/openshift-bundle/0.2.0/manifests/openvino-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (dtrawins) [reply extras/openvino-operator-openshift/openshift-bundle/0.2.0/manifests/openvino-operator.clusterserviceversion.yaml:230]: @mzegla it is intentional, otherwise we would need to create new offering in RH catalogue. It was agreed to do it for golang version later

File: extras/openvino-operator-openshift/openshift-bundle/0.2.0/metadata/annotations.yaml (hunk 3 above)
Comment (dtrawins) [reply extras/openvino-operator-openshift/openshift-bundle/0.2.0/metadata/annotations.yaml:6]: @mzegla it is intentional, otherwise we would need to create new offering in RH catalogue. It was agreed to do it for golang version later

**Review by mzegla — APPROVED** [review]
